doc/search.md

```diff
@@ -4,3 +4,56 @@ Search
 Different examples of search which have been implemented in the example [SearchExamplesAsAppUser.java](https://github.com/box/box-java-sdk/blob/master/src/example/java/com/box/sdk/example/SearchExamplesAsAppUser.java)
 
 For more information refer [search documentation](https://developer.box.com/v2.0/reference#searching-for-content)
+
+* [Search with Offset](#search-with-offset)
+* [Search with Limit](#search-with-limit)
+* [Search with Parameters](#search-with-parameters)
+
+Search with Offset
```

Comment:
Since these all look like they refer to the same method, maybe combine into a single section that explains all three params?

Reply:
@mattwiller, good point, will do!